src/lib/components/particles/waffle/index.jsx

```diff
@@ -0,0 +1,37 @@
+import defaultStyles from "./style.module.css"
+import { mergeStyles } from "$styles/helpers/mergeStyles"
+
+
+const renderUnit = (squares, attributes) => squares ? <rect {...attributes} /> : <circle {...attributes} />
+
+export const Waffle = ({ units, rows, total, squares = false, abbreviationAccessor = 'party', onMouseOver, paddingTop, showHalfLine, styles }) => {
+  //units is an array of objects with the following structure: { [abbreviationAccessor]: 'A' }.
+  const width = 1300
```

Comment:
I think components should be size-agnostic, i.e. resize to fit their container. See `useContainerSize` hook.